internal/testutils/database.go

```diff
@@ -147,3 +151,17 @@ func testDbString(dbname string) string {
 		config.Get().DatabasePort,
 	)
 }
+
+func ConnectAndMigrateDB(packageName string) {
+	ConnectToTestDB(packageName)
+
+	if out := dao.DB.Exec("CREATE SCHEMA " + packageName); out.Error != nil {
+		fmt.Println("error in creating schema " + out.Error.Error())
+	}
+
+	MigrateSchema()
+
+	if out := dao.DB.Exec("SET search_path TO \"$user\", " + packageName); out.Error != nil {
```

Comment:
You might find it easier using a raw string here, as you can avoid escaping double quotes:

```diff
- dao.DB.Exec("SET search_path TO \"$user\", " + packageName);
+ dao.DB.Exec(`SET search_path TO "$user", ` + packageName);
```